### PR TITLE
Enable routing of CSP assets

### LINKED ISSUE #8699: [PS AMP] Setup PS routes for STY/CSP routes in Simorgh
**Is your feature request related to a problem? Please describe.**
The following PS site will have their AMP pages rendered by Simorgh. To enable this we need to update the Simorgh react-router regex matchers to allow the PS sites.

```
news
sport
newsround
cymrufyw
naidheachdan
```

**Describe the solution you'd like**

#### Allow PS routes to work in Simorgh
Update the Simorgh route matches to enable the PS sites to have their AMP pages rendered in Simorgh.

The route matchers pull the list of services from the loadabe config.
We need to update the loadable config to include the PS service sites listed above. This requires that the service config files have already been created ( #8678 )

#### Update page type configurations
PS makes use of CSP pages, these are identical to STY pages and should use the same page type container as STY assets.

**Definition of done**
- The fixture data added in #8698 is routable in Simorgh and renders locally for each PS site

**Testing notes**
[Tester to complete]

### PR DESCRIPTION
**Pull request by chris-hinds**
Resolves #8699 

**Overall change:**
- Add a new CSP page type
- Ensure CSP assets render using the story page container

Example CSP asset http://localhost:7080/news/business-55345826

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [X] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1910,14 +1910,20 @@
       "integrity": "sha512-QvO1J9drc7K2SeBrHuO5/8nQ1QTks/OOG+vwva4lM7T28pPgf556snMpPE/3KE6uD2UJky8I4A5dNRNbqAXiDw==",
       "dev": true,
       "requires": {
-        "axios": "0.21.1",
+        "axios": "0.19.0",
```
Comment (andrewscfc) [inline package-lock.json]: you might wanna run npm install again, it seems to have dropped the force-resolutions :(

Comment (chris-hinds) [reply package-lock.json]: eughhhhh. Good spot @andrewscfc

Comment (chris-hinds) [reply package-lock.json]: hmm running npm install isnt changing anything, I think its working as expected as axios 0.21.1 is declared in the dependencies object below?

Comment (andrewscfc) [reply package-lock.json]: hmm so weird, this occasionally seems to switch formats `npx npm-force-resolutions` updates it to an invalid format, then `npm install` updates it again to a valid format, but not always in the same way :( 

Anyway nothing to do with this PR.

**Review by andrewscfc — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (chris-hinds) [reply package-lock.json]: I just ran `npx npm-force-resolutions` and that seemed to worked. Have updated this PR

**Review by simonsinclair — APPROVED** [review]

**Pull request merged**